.github/workflows/release-homebrew.yaml

```diff
@@ -0,0 +1,164 @@
+# When a third-party action is added (i.e., `uses`), please also add it to `download-licenses` in Makefile.
+# When the installers are ready in installer private bucket, run this workflow to test them.
+# TODO: Add job of uploading installers to Github release when installer tests are stable.
```

Comment:
Seems homebrew only need to make a pr instead of uploading the pkg?

Reply:
Will do.

Reply:
> Seems homebrew only need to make a pr instead of uploading the pkg?
I think the TODO item means we need a workflow to upload the pkg installer from s3 bucket to the release assets after the e2e tests passes. It's not for homebrew release. Does it make sense to you?